zzk/service/hoststate_test.go

```diff
@@ -33,27 +33,23 @@ var (
 	ErrTestNoAttach = errors.New("could not attach to container")
 )
 
+const (
+	servicePath string = "/services/serviceid"
+	hostPath    string = "/hosts/hostid"
+	hostId      string = "hostid"
+	serviceId   string = "serviceid"
+	serviceName string = "serviceA"
+	imageId     string = "imageid"
+	containerId string = "containerid"
+)
+
```

Comment:
So then we have to create new constants everytime we need to create new variables?  I don't agree that is a good practice.  Alternatively, I would be okay with declaring the constants at the beginning of the functions.